package.json

```diff
@@ -20,6 +20,10 @@
     "start": "yarn install && electron ./app",
     "pack": "electron-builder --dir",
     "dist": "electron-builder",
+    "dist:linux": "electron-builder --linux",
+    "dist:linux:appimage": "electron-builder --linux AppImage",
+    "dist:linux:snap": "electron-builder --linux snap",
+    "dist:macOS": "electron-builder --mac",
```

Comment:
We should probably not include the macOS into the build as there is a Microsoft Teams Mac client (https://teams.microsoft.com/downloads). It might bring more questions than anything.

Reply:
Hi Ismael, I just made the changes! Removed mac and added snap and AppImage to Linux.